### PR TITLE
Documented updates to feature flag list and table

### LINKED ISSUE #3546: Document updates to feature flag list and table
Reference: https://rancher.com/docs/rancher/v2.6/en/installation/resources/feature-flags/

### PR DESCRIPTION
**Pull request by jtravee**
Closes #3546

Not including Harvester (https://github.com/rancher/docs/pull/3534) and RKE2 (https://github.com/rancher/docs/pull/3545) for their feature flag updates.

### CODE REVIEW

File: content/rancher/v2.6/en/installation/resources/feature-flags/_index.md [hunk 1]
```diff
@@ -25,11 +25,13 @@ For example, if you install Rancher, then set a feature flag to true with the Ra
...
-- `istio-virtual-service-ui`: This feature enables a [UI to create, read, update, and delete Istio virtual services and destination rules]({{<baseurl>}}/rancher/v2.6/en/installation/resources/feature-flags/istio-virtual-service-ui), which are traffic management features of Istio.
-- `unsupported-storage-drivers`: This feature [allows unsupported storage drivers.]({{<baseurl>}}/rancher/v2.6/en/installation/resources/feature-flags/enable-not-default-storage-drivers) In other words, it enables types for storage providers and provisioners that are not enabled by default.
+- `fleet`: The previous `fleet` feature flag is now required to be enabled as the fleet capabilities are leveraged within the new provisioning framework. If you had this feature flag disabled in earlier versions, upon upgrading to Rancher, the flag will automatically be enabled. See this [page]({{<baseurl>}}/rancher/v2.6/en/deploy-across-clusters/fleet) for more information.
```
Comment (btat) [inline content/rancher/v2.6/en/installation/resources/feature-flags/_index.md:28]:
```suggestion
- `fleet`: The previous `fleet` feature flag is now required to be enabled as the Fleet capabilities are leveraged within the new provisioning framework. If you had this feature flag disabled in earlier versions, upon upgrading to Rancher v2.6, the flag will automatically be enabled. See this [page]({{<baseurl>}}/rancher/v2.6/en/deploy-across-clusters/fleet) for more information.
```

File: content/rancher/v2.6/en/installation/resources/feature-flags/_index.md [hunk 2]
```diff
@@ -45,7 +47,9 @@ The below table shows the availability and default value for feature flags in Ra
 | `fleet`  | `true` | GA* | v2.5.0 |   |
 | `fleet`  | `true` | Can no longer be disabled | v2.6.0 | N/A  |
 | `continuous-delivery` | `true` | GA* | v2.6.0 | |
-| `token-hashing` | `false` | GA* | v2.6.0 | |
+| `token-hashing` | `false` for new installs, `true` for upgrades | GA* | v2.6.0 | |
+| `legacy` | `false` for new installs, `true` for upgrades | GA* | v2.6.0 | |
+| `multi-cluster-management` | `false` | GA* | v2.x | |
```
Comment (btat) [inline content/rancher/v2.6/en/installation/resources/feature-flags/_index.md:52]: v2.x is too broad of a range. We should at least list the minor version.

**Review by btat — CHANGES_REQUESTED** [review]
Features that are no longer available should be removed from the table as well.

File: content/rancher/v2.6/en/installation/resources/feature-flags/_index.md (hunk 1 above)
Comment (jtravee) [reply content/rancher/v2.6/en/installation/resources/feature-flags/_index.md:28]: Made edits as recommended.

File: content/rancher/v2.6/en/installation/resources/feature-flags/_index.md (hunk 2 above)
Comment (jtravee) [reply content/rancher/v2.6/en/installation/resources/feature-flags/_index.md]: Updated to 2.5.0

**Comment by btat** [comment]
@jtravee your commit https://github.com/rancher/docs/pull/3548/commits/b8cc2cb2889ca217839676a49980e5ed1f915e46 added some unrelated (azure cloud provider) content to this PR

**Pull request merged**